.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -24,25 +24,30 @@ body:
     id: steps-to-reproduce
     attributes:
       label: Steps to Reproduce
-      description: "How can the issue be reliably reproduced? feel free to include screenshots or other supporting artifacts"
-      placeholder: |
+      description: |
+        How can the issue be reliably reproduced? Feel free to include screenshots or other supporting artifacts
+        
         Example:
         1. In the test environment, fill out the application for a new domain
         2. Click the button to trigger a save/submit on the final page and complete the application 
         3. See the error
+      value: |
+        1.
```

Comment:
In this case, we know only that all steps will start at `1.`, so we can provide that to save some clicks/typing